src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MoveQueryRepositoryTest.kt

```diff
@@ -76,58 +78,91 @@ internal class MoveQueryRepositoryTest {
       )
     )
 
-    moveRepository.save(standardMove)
+    priceRepository.save(
```

Comment:
This is duplicates the price but for GEOAmey to recreated and test the issue is resolved.